Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -4010,7 +4010,7 @@ final class WorkspaceTests: XCTestCase {
                     toolsVersion: .v5_2
                 ),
             ],
-            toolsVersion: .v5_2,
+            toolsVersion: ToolsVersion(version: "5.2.5"),
```

Comment:
I don't know why only this `TestWorkspace(toolsVersion:)` had to be changed and not all the `TestPackage(toolsVersion:)` above, haven't looked into it.